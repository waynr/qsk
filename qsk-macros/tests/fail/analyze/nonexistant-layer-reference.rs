use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer[Active]: {
            Y -> HOME,
            F -> TT(Meow, F),
        },
        Navigation: {
            END -> Exit(),
        },
    );
}
