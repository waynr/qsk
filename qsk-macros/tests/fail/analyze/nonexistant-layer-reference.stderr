error: layer reference does not exist

         = help: existing layers include: {"ModLayer", "Navigation"}

 --> tests/fail/analyze/nonexistant-layer-reference.rs:7:21
  |
7 |             F -> TT(Meow, F),
  |                     ^^^^
